lms/djangoapps/discussion_api/serializers.py

```diff
@@ -75,11 +77,20 @@ def validate_not_blank(value):
         raise ValidationError("This field may not be blank.")
 
 
+def get_user_profile_image_url(request, username):
+    """
+    Returns image urls against a user profile
+    """
+    user = User.objects.select_related('profile').get(username=username)
+    return AccountLegacyProfileSerializer.get_profile_image(user.profile, user, request)
```

Comment:
`get_profile_image` returns a dictionary of different image urls, can we change the name of fields, doc strings & methods in this file accordingly.